index.html

```diff
@@ -26,49 +26,289 @@
+              sign-ups required.
+            </p>
+          </div>
+          <div class="see-proj">
+            <a href="" class="lang">html</a>
+            <a href="" class="lang">css</a>
+            <a href="" class="lang js-anc">javaScript</a>
+            <button class="buttn">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="myWork myWork3">
+        <div class="snapshot-port">
+          <img src="/assests/images/Snapshoot Portfolio (2).png" alt="" />
+        </div>
+        <div class="left-block">
+          <div class="primary-text">
+            <h1>Tonic</h1>
+          </div>
+
+          <ul class="second-list">
+            <li class="canopy">CANOPY</li>
+            <li class="other-li">
+              <img src="/assests/images/Counter.png" alt="" /> &nbsp; Back End
+              Dev
+            </li>
+            <li class="other-li">
+              <img src="/assests/images/Counter.png" alt="" /> &nbsp; 2015
+            </li>
+          </ul>
+          <div class="work-statment">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="see-proj">
+            <a href="" class="lang">html</a>
+            <a href="" class="lang">css</a>
+            <a href="" class="lang js-anc">javaScript</a>
+            <button class="buttn">See Project</button>
+          </div>
+        </div>
       </div>
-      <a class="meet">LET'S CONNECT</a>
-      <div class="social-icons">
-        <ul class="social-icons-list">
-          <li>
-            <a href="#"><img src="assests/images/Vector.png" alt="" /></a>
-          </li>
-          <li>
-            <a href="#"><img src="assests/images/Vector (1).png" alt="" /></a>
-          </li>
-          <li>
-            <a href="#"><img src="assests/images/Vector (2).png" alt="" /></a>
-          </li>
-          <li>
-            <a href="#"><img src="assests/images/Vector (3).png" alt="" /></a>
-          </li>
-          <li>
-            <a href="#"
-              ><img src="/assests/images/Linkedin icon.png" alt=""
-            /></a>
-          </li>
-        </ul>
+      <div class="myWork myWork4">
+        <div class="snapshot-port">
+          <img
+            src="/assests/images/Snapshoot Portfolio (3).png"
+            alt=""
+            class="multi-post-img"
+          />
+        </div>
+        <div class="left-block left-block-multi">
+          <div class="primary-text">
+            <h1>Multi-Post Stories</h1>
+          </div>
+
+          <ul class="second-list">
+            <li class="canopy">CANOPY</li>
+            <li class="other-li">
+              <img src="/assests/images/Counter.png" alt="" /> &nbsp; Back End
+              Dev
+            </li>
+            <li class="other-li">
+              <img src="/assests/images/Counter.png" alt="" /> &nbsp; 2015
```

Comment:
Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image